README.md

```diff
@@ -1,5 +1,13 @@
 # FreeRTOS AWS Reference Integrations
 
+## Update (05/13/2021)
+
+In the next few months, we will have an official release of the AWS Reference Integration repository which brings in the 202012.01-LTS libraries. In this release, we will carry the LTS libraries and remove legacy libraries. The legacy libraries will continue to be accessible through the release tags, and AWS will continue to support Github issues on these libraries on a case by case basis.
```

Comment:
I think we should say  'remove legacy libraries from main branch' to be more specific.

Reply:
added the text